src/pages/transcripts.js

```diff
@@ -23,7 +23,7 @@ export default function Transcripts({ data }) {
             <nav>
               <ol>
                 {articles.map((article) => (
-                  <li key={article.title}>
+                  <li key={article.path}>
```

Comment:
It seems like both the title and path properties are defined. What's the advantage to using path instead? Is it more stable or unique?

Reply:
Since this isn't apparent in the UI, either seems fine to me. Titles seem more likely to collide than paths

Reply:
> Titles seem more likely to collide than paths

Yes, this was the intended fix. On dev, it keeps popping the warning of same key values.